### ISSUE TITLE
Copying anything (title/url/password) from KeepassXC does not work (not Wayland)

### ISSUE THREAD

**Issue by paulb39** [keepassxreboot/keepassxc#7064]
## Overview
I can't copy anything (username / password /url / title) from KeepassXC)

## Steps to Reproduce
1. Open KeePassCX 
2. Right click entry and copy something -> copy username / password / title / notes
3. Try to paste into another program (like atom)

## Expected Behavior
Text is copied over

## Actual Behavior
Text is not copied over

## Context
Every other post about this issue I can find is with using Wayland, I am not using Wayland.
Also the only other solutions I've seen are to install xsel and xdotool, I've installed them and it made no difference. 

KeePassXC - 2.6.6
Revision: 9c108b9
Distribution: Snap

Operating System: Ubuntu 20.04
Desktop Env: Gnome
Windowing System: X11

**Comment by droidmonkey** [comment]
Can you copy text in any application?

**Comment by paulb39** [comment]
I cannot, tried text editors / browsers, even KeepassXC itself does not allow copying (like say try copying the url and adding it to notes, doesn't work).
Edit: oh sorry I think I misunderstood what you meant - copying works for other applications, like I can copy from firefox and paste it into atom just fine. I just can't copy anything from KeepassXC

**Comment by paulb39** [comment]
I see you added non reproducible - is there any other information about my system I can give you to help try to track this down?
I also see https://github.com/keepassxreboot/keepassxc/issues/1800 is still an open item - is there no verbose mode for keepassxc? so I can see if it throws any type of error when I try to copy in the program.

**Comment by droidmonkey** [comment]
Run the program from a terminal and see if you see any error messages. It is impossible to reproduce your exact environment. This is not reproducible at all in a stock configuration.

**Comment by paulb39** [comment]
So this is interesting...
I have keepassXC start on startup:
![image](https://user-images.githubusercontent.com/5777695/138288606-b515ed70-612a-4728-b004-fd1c45a78853.png)

On startup I have the copy issue. I tried running in terminal and copy works. So I then tried starting it via alt + f2 and that works as well. 

I can reproduce it every time - doesn't work on start up, works if I close it and open it again. Have any idea how I can diagnose why that is?
